### ISSUE TITLE
Review User Import

### ISSUE THREAD

**Issue by artlogic** [arsmentis/wdet-conversion#40]
Users were imported from the existing site - some of these users may not be needed but may be tied to content, so they've all been imported.

https://wdetcms.wdet.org/wp-admin/users.php

**Comment by metastange** [comment]
Ok so if I remove Amy Miller (user) that should not impact the content created by Amy Miller (author)? Just want to double check before I start scrubbing.

**Comment by artlogic** [comment]
> Are users people who should have active access to the site and authors people who are tied to content? If so, should i remove non-active user profiles?

Users represent the list of all users that created content on the old site - including inactive ones. You'll notice everyone except a few people are set to subscribers which means they effectively have no access. You can see for yourself by changing the email address to yours, requesting a password reset and seeing what that user can do.

If you remove those users, internally wordpress will reassign that content back to the admin account, including information on who uploaded media and who originally created posts. If that's not an issue then we can remove many of the users. It would be best to do it automatically on import based on some criteria from the old CMS.

Authors are an extension that essentially extend the limited concept of users in wordpress. They are used only to communicate authorship information of a post, and to create author profiles. They can exist without associated users, but can't create content or access the site in any way.

> Ok so if I remove Amy Miller (user) that should not impact the content created by Amy Miller (author)? Just want to double check before I start scrubbing.

If you plan to scrub users, do it on the old CMS. You can experiment with doing it in wdetcms - but we're not at the final import yet, so I wouldn't do anything that's a lot of work as it'll be lost.

**Comment by metastange** [comment]
@artlogic If I provide a list of usernames, are you able to run a query on site content to see if there is content associated with that user, like you did for Tia Graham last week? That would help me determine who can be removed entirely and who has bylines that need to remain intact.

**Comment by metastange** [comment]
See attached for full list.

[wdet user content verification.xlsx](https://github.com/arsmentis/wdet-conversion/files/7367593/wdet.user.content.verification.xlsx)
.

**Comment by metastange** [comment]
Got it, no need to apologize, I'd rather understand this and work through it now than be surprised by something later!

We created a test user (CMSTest21) and created an unpublished test post (CMS Test A). Tony then deleted the user which, it appears on my end, also eliminated the post.

To that end, I will not remove users on the current CMS in order to ensure the content remains intact. Once we've transitioned to WordPress, I'll remove non-active users and the content will be attributed to admin.

One question I had is the post list you provided is based on people IDs/bylines - is there any way to tell how many posts are tied to users if the post content is user-reliant, not person/byline reliant?
